pkg/plugins/policies/core/matchers/testdata/matchedpolicies/fromrules/select-single-inbound.policies.yaml

```diff
@@ -0,0 +1,32 @@
+type: MeshTrafficPermission
+mesh: mesh-1
+name: mtp-1
+spec:
+  targetRef:
+    kind: Dataplane
+    name: dp-1
+    sectionName: main-port
```

Comment:
You can also add test when `mtp-1` selects the entire DPP and `mtp-2` selects only one inbound. This reminds me about order, do we need to update sorting as well?

Reply:
updated sorting, and added tests for it